bin/run.sh

```diff
@@ -34,7 +34,13 @@ if [ -z "$FLUO_HOME" ]; then
   exit 1
 fi
 
+maven_opts=
+if [ -n "$FLUO_MAVEN" ]; then
```

Comment:
Could call this PHRASE_COUNT_MAVEN instead.  FLUO_MAVEN implies this is something the fluo scripts respect.

Reply:
How about `MAVEN_SETTINGS`? I just want something generic that works across multiple projects.

Reply:
I removed the maven config as it's possible to just created an alias for mvn and set a config file